### PR TITLE
Restrict user signup email domain

### PR DESCRIPTION
**Pull request by ruddfawcett**
As per discussion in #42, this adds (and documents) the ability to restrict the email domain from with which users can sign up.

### CODE REVIEW

File: app/controllers/invite_controller.rb [hunk 1]
```diff
@@ -20,6 +20,15 @@ def submit
     first_name = params[:first_name]
     last_name = params[:last_name]
 
+    if ENV["RESTRICTED_DOMAIN"]
+      if email.split("@").last != ENV["RESTRICTED_DOMAIN"]
+        @message = "Sorry! Early access is currently restricted to people within the #{ENV["RESTRICTED_DOMAIN"]} domain.  We will be opening it up soon, though!"
```
Comment (KrauseFx) [inline app/controllers/invite_controller.rb]: I'd remove the `  We will be opening it up soon, though!` as it might not be correct for everyone.

**Comment by KrauseFx** [comment]
This looks like a great addition, thanks for the pull request. I added one comment before it's ready to be merged :+1:

File: app/controllers/invite_controller.rb (hunk 1 above)
Comment (ruddfawcett) [inline app/controllers/invite_controller.rb]: @KrauseFx Oops -- you're right.  This was copied from my fork verbatim. I'll change that.

**Comment by KrauseFx** [comment]
Thank you so much @ruddfawcett :+1:

**Pull request merged**